test/OpenTelemetry.Extensions.Hosting.Tests/OpenTelemetryServicesExtensionsTests.cs

```diff
@@ -326,6 +327,129 @@ public void AddOpenTelemetry_WithMetrics_NestedResolutionUsingConfigureTest()
         Assert.True(innerTestExecuted);
     }
 
+    [Fact]
+    public void AddOpenTelemetry_WithLogging_SingleProviderForServiceCollectionTest()
+    {
+        var services = new ServiceCollection();
+
+        services.AddOpenTelemetry().WithLogging(builder => { });
+
+        services.AddOpenTelemetry().WithLogging(builder => { });
+
+        using var serviceProvider = services.BuildServiceProvider();
+
+        Assert.NotNull(serviceProvider);
+
+        var tracerProviders = serviceProvider.GetServices<LoggerProvider>();
```

Comment:
nit: 
```suggestion
        var =loggerProviders = serviceProvider.GetServices<LoggerProvider>();
```

Reply:
Thanks I will fix. Also noticed we don't have tests for `OpenTelemetryBuilder.ConfigureResource` I'll add one/some.